.github/workflows/production.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy prod
+run-name: ${{ github.actor }} is deploying 🚀
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  test-and-deploy:
+    runs-on: ubuntu-latest
+    environment: Production
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - run: npm ci
+      - run: mkdir -p ~/.local/bin
+      - run: curl -L https://github.com/apache/incubator-openwhisk-cli/releases/download/latest/OpenWhisk_CLI-latest-linux-amd64.tgz | tar xz -C ~/.local/bin
+      - run: npm run lint
+      - run: npm run test:coverage
+      - shell: bash
+        env:
+          ADOBE_RUNTIME_AUTH: ${{ secrets.ADOBE_RUNTIME_AUTH }}
+        run: |
+          bash deploy.sh lookup production && bash deploy.sh setgithubcheck production && bash deploy.sh checker production && bash deploy.sh signwebhook production
+      # - run: npm run test:integration, add this back later
```

Comment:
When's the "add this back later"? Also, should this be moved up to after the `test:coverage`?

Reply:
no, this tests the deployed copy to make sure everything worked, however, I can't do the integration tests at all right now, see https://github.com/adobe/cla-bot/pull/163#issue-2532555773
i'd love to actually get the integration tests running again some day, so I left them in the repo and where they would be expected to be run in the flow